package.json

```diff
@@ -1,26 +1,37 @@
 {
   "devDependencies": {
+    "babel-eslint": "8.0.2",
+    "eslint-config-prettier": "2.7.0",
+    "eslint-plugin-prettier": "2.3.1",
+    "eslint-plugin-react": "7.4.0",
+    "eslint": "4.11.0",
     "husky": "0.14.3",
     "lerna": "2.5.1",
-    "lint-staged": "4.3.0",
-    "prettier": "1.8.1"
+    "lint-staged": "5.0.0",
+    "prettier": "1.8.2"
   },
   "lint-staged": {
-    "*.{js,ts}": [
-      "yarn lint:script",
-      "git add"
-    ]
+    "*.{js,jsx}": ["eslint --ignore-path .prettierignore --fix", "git add"],
+    "*.json": ["prettier --write", "git add"],
+    "*.md": ["prettier --write", "git add"],
+    "*.scss": ["prettier --write", "git add"],
+    "*.ts": ["prettier --write", "git add"]
   },
   "private": true,
   "scripts": {
     "boot": "lerna bootstrap",
     "clear": "lerna run clear",
     "dist": "lerna run dist",
-    "lint": "yarn lint:script",
-    "lint:script": "prettier --ignore-path .gitignore --write **/*.{js,ts}",
+    "fix:config": "prettier --write '**/*.json'",
+    "fix:docs": "prettier --ignore-path .gitignore --write '**/*.md'",
+    "fix:jsScript": "npm run test:jsScript -- --fix",
+    "fix:script": "yarn fix:jsScript && yarn fix:tsScript",
+    "fix:style": "prettier --write '**/*.scss'",
+    "fix:tsScript": "prettier --write '**/*.ts'",
+    "fix": "yarn fix:docs && yarn fix:config && yarn fix:style && yarn fix:script",
```

Comment:
another idea is to have one `fix:stuff: "prettier --ignore-path .gitignore --write '**/*.{md,json,scss,ts}'"` and `fix:eslint` for the `{js,jsx}` because of the ESLint.. to simplify it..